tests/e2e/standalone/commands.go

```diff
@@ -137,6 +137,11 @@ func cmdStop(appId string, args ...string) (string, error) {
 	return spawn.Command(common.GetDaprPath(), stopArgs...)
 }
 
+func cmdStopWithRunTemplate(runTemplateFile string, args ...string) (string, error) {
+	stopArgs := append([]string{"stop", "--log-as-json", "-f", runTemplateFile}, args...)
+	return spawn.Command(common.GetDaprPath(), stopArgs...)
+}
+
```

Comment:
refactor `cmdStop` and `cmdStopRunTemplate` into a single function and then have both `cmdStop` and `cmdStopRunTemplate` use that refactored function ...